sourcegraph/about
# Update ATC event page copy

Replace Emily's speaker profile with Max's.

### Changelog

- Closes #5677



### Test

1. Ensure prettier has standardized the proposed changes.
2. Verify the new copy matches what was in Figma
